PR: https://github.com/odoo/odoo/pull/200337

From: 12090f6b8552aeacf86c9ec9462c76041fee2d4c
From: kinhosz
Date: 2025-04-14 15:05:43

Breaking data model changes scores: del:3 + add:0, change matches:
-    module_delivery_fedex = fields.Boolean("FedEx Connector")
-    module_delivery_ups = fields.Boolean("UPS Connector")
-    module_delivery_usps = fields.Boolean("USPS Connector")

Total Changes: 6

[IMP] stock,{website_}sale: install shipping connectors rest API

The new REST API versions of the shipping connectors (FedEx, UPS, USPS)
have been released. However, when installing one from the settings, the
system still installs the legacy modules.
This commit enables the new REST API versions instead of the legacy ones.
It applies to FedEx, UPS, and USPS connectors in the following settings:
* Sales -> Shipping
* Inventory -> Shipping Connectors
* eCommerce -> Delivery

closes odoo/odoo#200337

Task: 4613485
Related: odoo/enterprise#80762
Related: odoo/upgrade#7334
Signed-off-by: Arnold Moyaux (arm) <arm@odoo.com>

================================= pseudo patch: =================================

--- a/addons/stock/models/res_config_settings.py
+++ b/addons/stock/models/res_config_settings.py
@@ -32,9 +32,9 @@ class ResConfigSettings(models.TransientModel):
     module_stock_sms = fields.Boolean("SMS Confirmation")
     module_delivery = fields.Boolean("Delivery Methods")
     module_delivery_dhl = fields.Boolean("DHL Express Connector")
-    module_delivery_fedex = fields.Boolean("FedEx Connector")
-    module_delivery_ups = fields.Boolean("UPS Connector")
-    module_delivery_usps = fields.Boolean("USPS Connector")
+    module_delivery_fedex_rest = fields.Boolean("FedEx Connector")
+    module_delivery_ups_rest = fields.Boolean("UPS Connector")
+    module_delivery_usps_rest = fields.Boolean("USPS Connector")
     module_delivery_bpost = fields.Boolean("bpost Connector")
     module_delivery_easypost = fields.Boolean("Easypost Connector")
     module_delivery_sendcloud = fields.Boolean("Sendcloud Connector")
